src/storage/sftp/storage.c

```diff
@@ -1158,7 +1195,11 @@ storageSftpNew(
             LIBSSH2_KNOWNHOSTS *const knownHostsList = libssh2_knownhost_init(this->session);
 
             if (knownHostsList == NULL)
-                THROW_FMT(ServiceError, "failure during libssh2_knownhost_init");
+            {
+                THROW_FMT(
+                    ServiceError,
+                    "failure during libssh2_knownhost_init: libssh2 errno [%d]", libssh2_session_last_errno(this->session));
```

Comment:
Don't we want the error text here as in other places?

Reply:
Updated to utilize storageSftpLibSsh2SessionLastError() to add libssh2 generated error message to the service error